tracer/src/Datadog.Trace/Ci/IntelligentTestRunnerClient.cs

```diff
@@ -923,6 +961,24 @@ private async Task<string> GetBranchNameAsync()
         return gitOutput;
     }
 
+    private readonly struct SearchCommitResponse
+    {
+        public readonly string[] LocalCommits;
+        public readonly string[] RemoteCommits;
+        public readonly bool IsOk;
+
+        public SearchCommitResponse(string[]? localCommits, string[]? remoteCommits, bool isOk)
+        {
+            LocalCommits = localCommits ?? Array.Empty<string>();
+            RemoteCommits = remoteCommits ?? Array.Empty<string>();
+            IsOk = isOk;
+        }
+
+        public bool HasCommits => LocalCommits.Length > 0;
+
+        public string[] MissingCommits => LocalCommits.Except(RemoteCommits).ToArray();
```

Comment:
Do we need any trimming of either of these values to be safe, or are we happy enough? I assume it's fine 🙂